modules/wallet/update.go

```diff
@@ -317,7 +294,7 @@ func (w *Wallet) ProcessConsensusChange(cc modules.ConsensusChange) {
 	if err := w.revertHistory(w.dbTx, cc.RevertedBlocks); err != nil {
 		w.log.Println("ERROR: failed to revert consensus change:", err)
 	}
-	if err := w.applyHistory(w.dbTx, cc.AppliedBlocks); err != nil {
```

Comment:
should be able to drop a quick function here to count the blocks, that will give you an accurate height for the wallet I think.